doc/en/warnings.rst

```diff
@@ -78,6 +79,16 @@ all other warnings into errors.
     filterwarnings =
         error
         ignore::UserWarning
+        ignore:function ham\(\) is deprecated:DeprecationWarning
+
+.. code-block:: toml
+
+    [tool.pytest.ini_options]
+    filterwarnings = [
+        "error",
+        "ignore::UserWarning",
+        'ignore:function ham\(\) is deprecated:DeprecationWarning',
```

Comment:
I'd suggest adding a comment about why single quotes vs double quotes matters here. :)

Reply:
I don't know how best to phrase it, can you make a `` ```suggestion ``, please?

Reply:
Sorry, I don't have a good suggestion as to the exact phrase either.